### PR TITLE
Fix/dapp staking animation

### PR DESCRIPTION
**Pull request by gtg7784**
**Pull Request Summary**

> Refactor to not use iframe for animation

**Check list**

- [ ] contains breaking changes
- [ ] adds new feature
- [x] modifies existing feature (bug fix or improvements)
- [ ] relies on other tasks
- [ ] documentation changes
- [ ] tested on mobile devices

**This pull request makes the following changes:**

**Fixes**

- Refactor VoteSTakeButtonBg component to not use iframe

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -82,6 +82,7 @@
     "store": "^2.0.12",
     "stream": "^0.0.2",
     "swiper": "^11.0.3",
+    "three": "^0.165.0",
```
Comment (bobo-k2) [inline package.json:85]: Why is this needed?

File: src/staking-v3/components/VoteStakeButtonBg.vue [hunk 2]
```diff
@@ -1,16 +1,243 @@
...
+    return vec3(srgb2rgb(srgb.r + SRGB_EPSILON), 
+                srgb2rgb(srgb.g + SRGB_EPSILON), 
+                srgb2rgb(srgb.b + SRGB_EPSILON));
+}
+vec4 srgb2rgb(vec4 srgb) {
+    return vec4(srgb2rgb(srgb.rgb), srgb.a);
+}
+#endif
+/*
+original_author: Patricio Gonzalez Vivo
```
Comment (bobo-k2) [inline src/staking-v3/components/VoteStakeButtonBg.vue:51]: Is it enough to state authors to use this code on the portal? Can you check the license?

**Review by bobo-k2 — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (gtg7784) [reply package.json:85]: this three dependency is for the creating 3D animation with WebGL

File: src/staking-v3/components/VoteStakeButtonBg.vue (hunk 2 above)
Comment (gtg7784) [reply src/staking-v3/components/VoteStakeButtonBg.vue:51]: this `fragmentShader` and `vertexShader` is from original iframe file, so I think it's better to keep it

**Review by bobo-k2 — APPROVED** [review]
Nice job. Thank you

**Review by impelcrypto — APPROVED** [review]
LGTM
